_docs/samples/bookinfo.md

```diff
@@ -87,14 +87,26 @@ reviews-v1-874083890-f0qf0                  2/2       Running   0          6m
 reviews-v2-1343845940-b34q5                 2/2       Running   0          6m
 reviews-v3-1813607990-8ch52                 2/2       Running   0          6m
 ```
-1. Determine the ingress' URL:
+1. Determine the gateway ingress URL:
+
+  If your cluster is running in an environment that supports external loadbalancers,
+use the ingress' external address:
```

Comment:
This change is not needed. The ADDRESS field in the ingress resource will always be set to the correct value. @ijsnellf could you please confirm?